Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL authors="Anubhav Mishra, Luke Kysow"
 ENV DEFAULT_TERRAFORM_VERSION=0.13.3
 
 # In the official Atlantis image we only have the latest of each Terraform version.
-RUN AVAILABLE_TERRAFORM_VERSIONS="0.8.8 0.9.11 0.10.8 0.11.14 0.12.29 0.13.5 ${DEFAULT_TERRAFORM_VERSION}" && \
+RUN AVAILABLE_TERRAFORM_VERSIONS="0.8.8 0.9.11 0.10.8 0.11.15 0.12.31 0.13.7 ${DEFAULT_TERRAFORM_VERSION}" && \
```

Comment:
Should we be worried that the DEFAULT_TERRAFORM_VERSION still pulls in a vulnerable 0.13.3?